mlflow/tracking/fluent.py

```diff
@@ -302,8 +297,21 @@ def _get_experiment_id_from_env():
     return env.get_env(_EXPERIMENT_ID_ENV_VAR)
 
 
+def _get_or_create_experiment(experiment_name):
+    client = MlflowClient()
+    exp = client.get_experiment_by_name(experiment_name)
+    exp_id = exp.experiment_id if exp else None
+    if exp_id is None:
+        print("INFO: '{}' does not exist. Creating a new experiment".format(experiment_name))
+        client.create_experiment(experiment_name)
+        return client.get_experiment_by_name(experiment_name)
```

Comment:
Recommend changing this to ...

```
exp_id = client.create_experiment(experiment_name)
return client.get_experiment(exp_id)
```

In `FileStore` implementation looking up based on name would be slightly expensive.